### PR TITLE
Changed README for release

### PR DESCRIPTION
**Pull request by jgraef**
Please review if the wording is okay.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,24 +33,71 @@ Besides [Rust](https://www.rust-lang.org/learn/get-started#installing-rust) itse
 - `pkg-config`
 - `libssl-dev` (in Debian/Ubuntu) or `openssl-dev` (in Fedora/Red Hat)
 
+### From crates.io
+
+To download from [crates.io](https://crates.io), compile and install the client:
+
+```bash
+cargo install nimiq-client
```
Comment (paberr) [inline README.md]: Does that use nightly automatically?

**Review by paberr — COMMENTED** [review]
Apart from that it sounds good

File: README.md (hunk 1 above)
Comment (jgraef) [reply README.md]: That was actually still on my TODO list. I think you have to explicitly override it. I will fix this later.

**Comment by jgraef** [comment]
I will also change some parts regarding the configuration, since I think there should be no default configuration and users should really copy and edit the example config. What do you think about that?

Another change about configuration: By default it will be in `$HOME/.nimiq/client.toml`. This way it doesn't matter where you run `nimiq-client` and we have a centralized location to store Nimiq configuration files, database and peer key file.

**Pull request merged**